packages/base-map/src/index.tsx

```diff
@@ -143,6 +150,24 @@ const BaseMap = ({
           <ul
             className={`layers-list ${fakeMobileHover && "fake-mobile-hover"}`}
           >
+            {typeof baseLayer === "object" &&
+              baseLayer.map((layer: string, index: number) => {
+                return (
+                  <li key={index}>
+                    {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+                    <label>
```

Comment:
Go ahead and add the `htmlFor` prop to the label, even if it already contains the input control, and delete the a11y exception.

Reply:
(Might do that for the checkbox labels below too)

Reply:
Why are we changing this now? We've been using this pattern everywhere in the project so far